public/setup/atomicupdate/symfonize_stats_position.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$sql[]="DELETE` FROM {$dbprefix}acces` WHERE `page`='statistiques/postes.php' ;";
+
```

Comment:
revert quote misplaced
<pre>$sql[]="DELETE FROM `{$dbprefix}acces` WHERE `page`='statistiques/postes.php' ;";</pre>

Reply:
solved with this commit 1b606b802939cc354ec73244cee8708f259c99ff